MapsetVerifier.Parser/Objects/HitObjects/Taiko/TaikoExtensions.cs

```diff
@@ -2,8 +2,100 @@
 {
     public static class TaikoExtensions
     {
-        public static bool IsDon(this Circle circle) => circle.hitSound != HitObject.HitSounds.Clap && circle.hitSound != HitObject.HitSounds.Whistle;
+        /// <summary>
+        ///     Returns whether the hit object is a don hit.
+        /// </summary>
+        public static bool IsDon(this HitObject hitObject)
+        {
+            if (hitObject is not Circle)
+            {
+                return false;
+            }
```

Comment:
```suggestion
```

Makes sense at first, but when you think about it more - this is basically saying to treat all non-circles as kat, which is kinda irrelevant since we treat dons and kats as equals. So we could just delete this.